sdk/keyvault/azure-security-keyvault-secrets/inc/azure/keyvault/secrets/keyvault_backup_secret.hpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief Keyvault BackupSecretData definition
+ */
+#pragma once
+#include <stdint.h>
+#include <vector>
+
+namespace Azure { namespace Security { namespace KeyVault { namespace Secrets {
+  /**
+   * @brief Represents a backed up secret.
+   *
+   */
+  struct BackupSecretResponse final
```

Comment:
I thought you typically suffix with "Result". Compare with other models to be consistent.

Reply:
done